cms/static/js/spec/views/programs/program_creator_spec.js

```diff
@@ -122,6 +123,25 @@ define([
                 expect( view.saveError ).not.toHaveBeenCalled();
             });
 
+            it( 'should submit the form correctly when creating micromasters program ', function(){
+                var programId = 221,
+                    micromasterProgramInput = $.extend({}, sampleInput);
+                
+                micromasterProgramInput.category = 'micromasters';
+                completeForm( sampleInput );
+
+                spyOn( $, 'ajax' ).and.callFake( function( event ) {
+                    event.success({ id: programId });
+                });
+
+                view.$el.find('.js-create-program').click();
+
+                expect( $.ajax ).toHaveBeenCalled();
+                expect( view.saveSuccess ).toHaveBeenCalled();
+                expect( view.goToView ).toHaveBeenCalledWith( String( programId ) );
+                expect( view.saveError ).not.toHaveBeenCalled();
```

Comment:
@schenedx consider moving this logic to a function so it's not duplicating as much code from the test above.